### PR TITLE
Add seed for a user and a workshop

### PR DESCRIPTION
**Pull request by tbutterwith**
These are small seeds that add a very simple workshop, and attach it to a user. I found these made developing a little easier as you don't have to create a user every time you want to log in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,8 +23,19 @@ To get this site running locally:
...
-### Running
+### Running the application
 
 Run `bundle exec rails server` and access `http://localhost:3000` on your browser of choice.
+
+### Running the tests
+
+Tests are found in `./spec` and can be run with `bundle exec rspec ./spec`.
+
+To run an individual test file `bundle exec rspac ./spec/path/to/file`
```
Comment (JiggyPete) [inline README.md]: Tiny Typo
```suggestion
To run an individual test file `bundle exec rspec ./spec/path/to/file`
```

File: README.md [hunk 2]
```diff
@@ -23,8 +23,19 @@ To get this site running locally:
...
+
+### Seeding the database
+
+To add sample data to the database, run `rake db:seed`. This is slightly different to the setup step as it does not clear and recreate the tables first.
+To create a fresh release of the database, run `rake db:setup`.
```
Comment (JiggyPete) [inline README.md]: I didn't know that's what `db:setup` did. 🎩

Interestingly it's always been one of the conversations about what should / should not be done in seeds (clearing them down on a proceeding run)

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,30 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
```
Comment (JiggyPete) [inline db/seeds.rb]: **Change request:** Delete the comments

Comment (tbutterwith) [reply db/seeds.rb]: I've dropped the example seeds from the comments but I think the information about how the seeds file works is worth keeping, seeing as there are a limited number of seeds there at the moment.

Comment (JiggyPete) [reply db/seeds.rb]: 👍

**Pull request merged**